db/migrate/20150218110610_create_project_instances.rb

```diff
@@ -0,0 +1,34 @@
+class CreateProjectInstances < ActiveRecord::Migration
+  def change
+    create_table :project_instances do |t|
+      t.references :project, index: true
+      t.string :name
+      t.string :url
+      t.text :backup_information
+      t.string :stage
+      t.string :branch
+      t.text :description
+      t.timestamps
+    end
```

Comment:
let's add a not null constraint to the following fields: project_id, name, url, stage, and also to project_instance_id in installations